src/NumericalAlgorithms/SphericalHarmonics/StrahlkorperFunctions.hpp

```diff
@@ -397,4 +397,47 @@ void time_deriv_of_strahlkorper(
     gsl::not_null<Strahlkorper<Frame>*> time_deriv,
     const std::deque<std::pair<double, Strahlkorper<Frame>>>&
         previous_strahlkorpers);
+
+/// @{
+/*!
+ * \brief Compute the power monitor of a Strahlkorper
+ *
+ * \details Computes a Strahlkorper represented as a spectral expansion
+ * in scalar spherical harmonics \f$Y_{lm}(\theta,\phi)\f$ up to
+ * \f$l=l_{\rm max}\f$ and \f$m=m_{\rm max}\f$. Then this function
```

Comment:
Use the more modern method for equations in dox, which is just `$Y_{lm}(\theta,\phi)$` (no `\f`)

Reply:
I did not know this was a thing...thanks! I was following [our instructions for writing good documentation](https://spectre-code.org/writing_good_dox.html#autotoc_md213) which don't yet have these improvements